effects/data_base_effect.py

```diff
@@ -0,0 +1,71 @@
+from typing import List
+
+import message_text_config as config
+import user_interaction
+import utils
+from effects.effect import Effect
+from message_text_config import Errors
+
+
+class DataBase(Effect):
+    def __init__(self, game: Game, name: str, creator: Citizen) -> None:
+        super(DataBase, self).__init__(game, name, creator, 4)
+
+    def _activate_impl(self):
+        target_numbers = []
+        for _ in range(3):
+            target_number = user_interaction.read_index("config.type_target")
+            target_numbers.append(target_number)
+
+        while (not self._validate(target_numbers)):
+            target_number = user_interaction.read_index("config.type_target")
+            target_numbers = []
+            for _ in range(3):
+                target_number = user_interaction.read_index(
+                    "config.type_target")
+                target_numbers.append(target_number)
+
+        self._targets = [
+            self._game.citizens[target_number - 1]
+            for target_number in target_numbers
+        ]
+        return True
+
+    def _validate(self, target_numbers: List[int]):
```

Comment:
SHOULD BE DISCUSSED: DB should not allow choosing same target more then one time (?)

Reply:
What do you mean?

Reply:
Done. Added comment before this check